pyproject.toml

```diff
@@ -79,9 +79,14 @@ test-command = "pytest --pyargs shapely.tests"
 source = ["shapely"]
 omit = ["shapely/tests/*"]
 
+[tool.doc8]
+include = ["docs"]
+ignore = ["D000", "D004"]
+allow-long-titles = true
```

Comment:
Also add `max-line-length = 79` to enforce this rule.

Reply:
This is the default, so was already enforced... but "explicit is better than implicit" so I added it...